frontend/document/koptinterface.lua

```diff
@@ -1239,6 +1238,26 @@ function KoptInterface:getPageBoxesFromPositions(doc, pageno, ppos0, ppos1)
     end
 end
 
+--[[--
+Compare positions within one page.
+Returns 1 if positions are ordered (if ppos2 is after ppos1), -1 if not, 0 if same.
+Actually positions of the word boxes containing ppos1 and ppos2 are compared.
```

Comment:
Do you mean?
`Current positions ...` (or `Actual positions...`, but I don't read this often in English)
or
`Actually, positions....` (adding some explanation, with a comma to make it less confusing)